Cargo.toml

```diff
@@ -37,6 +37,14 @@ exclude = [
     "test_enclave",
 ]
 
+# We need to explicitly specify resolver 2.
```

Comment:
The cake is a lie

I'm hoping someone might be able to prove me wrong here, but it seems to break without this.

Reply:
The relevant Cargo issue here is https://github.com/rust-lang/cargo/issues/4866 (and yes, `resolver = "2"` is required).

Reply:
Does `getrandom` use `edition = "2021"`?

Reply:
No, I added 2021 to `rand`, `getrandom`, and `rand_core` and it still resulted in the same error
```shell
error[E0463]: can't find crate for `std`
 --> /home/nick/.cargo/registry/src/github.com-1ecc6299db9ec823/getrandom-0.2.7/src/error_impls.rs:9:1
  |
9 | extern crate std;
  | ^^^^^^^^^^^^^^^^^ can't find crate
  |
  = note: the `thumbv7m-none-eabi` target may not support the standard library
  = help: consider building the standard library from source with `cargo build -Zbuild-std`
```